.github/workflows/image-related-on-push-scheduled.yml

```diff
@@ -0,0 +1,70 @@
+name: "Image and Trivy related tasks on push and scheduled"
```

Comment:
i would still name the file "image-publish ..." and the name "Publish Image ..."